app/assets/javascripts/fae/form/_ajax.js

```diff
@@ -285,6 +289,19 @@ Fae.form.ajax = {
     });
   },
 
+  fileDeleteLinks: function() {
+    $('body' ).on('click', 'a.js-file-clear', (function(e) {
```

Comment:
Use `FCH.$body`

Reply:
Is there any reason for wrapping the third argument (the anonymous function) in an uninvoked manner like this? Legitimate question, I've only seen `(function` when it's an IIFE.

Reply:
And - avoid [qualifying elements](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#qualifyingelement) like `a.`